nix-install.sh

```diff
@@ -106,9 +79,9 @@ main() {
     done
 
     if $_ansi_escapes_are_valid; then
-        printf "\33[1minfo:\33[0m downloading installer\n" 1>&2
+        printf "\33[1minfo:\33[0m downloading installer \33[4m$_url\33[0m\n" 1>&2
     else
-        printf '%s\n' 'info: downloading installer' 1>&2
+        printf '%s\n' 'info: downloading installer $_url' 1>&2
```

Comment:
```suggestion
        printf 'info: downloading installer (%s)\n' "$_url" 1>&2
```

$_url won't interpolate in single quotes, and we should use printf for the purpose anyway :)